privacy-policy.html

```diff
@@ -1,547 +1,555 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-  <title>LockState Privacy Policy</title>
-  <link rel="shortcut icon" href="https://lockstate-uploads.s3.amazonaws.com/production/store/site_favicon/20/favicon.ico?v=113" />
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <title>LockState Terms of Service</title>
+  <link rel="shortcut icon" href="https://connect.remotelock.com/favicon.ico" />
 
   <style type="text/css">
     body {
-      margin: 10px;
-      padding: 10px;
+      margin: 0;
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
Would be nice to add `system-ui` as first option. https://caniuse.com/#search=system-ui